modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+For more information see xref:content-appearance.adoc#premium-upgrade-promotion-option[Premium upgrade promotion option].
+
+=== New `format_noneditable_selector` option to specify the `contenteditable="false"` elements that can be wrapped in a format.
+
+Elements with `contenteditable="false"` attributes can now have formatting applied to them.
+
+And this new capability is configurable using the new `format_noneditable_selector` option.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  format_noneditable_selector: 'span,p.special'
+});
+----
+
+For details see xref:content-formatting.adoc#format_noneditable_selector[Content Formatting].
+
+=== Added `allow` as a valid attribute for the `iframe` element in the editor schema.
+
+Previously, the `allow` attribute was not considered a valid attribute for `iframe` elements in the {productname} schema. It now is.
+
+When a {productname} editor with an `allow` attribute —
+
+[source,html]
+----
+<iframe allow="fullscreen" style="width: 200px; height: 200px; border: 1px solid blue"></iframe>
+----
+
+— is initialized, the `allow` attribute will still be present in the rendered `iframe` element.
+
+=== MenuButtons can now contain a search field
+
+MenuButtons, buttons on the {productname} toolbar that show a menu when clicked, can now present a search field at the top of the menu.
+
+The button refetches items as the search field is updated.
+
+The new xref:mergetags.adoc[Merge Tags] Premium plugin includes a MenuButton with this new search field.
+
+[[improvements]]
+== Improvements
+
+{productname} 6.2 also includes the following improvements:
+
+=== Formatting can now be applied to an element that has a `contenteditable="false"` attribute
+
+Elements with `contenteditable="false"` attributes can now have formatting applied to them.
+
+For example, such elements can now be set to *bold* or _italic_. As well, such elements can have their font and font-size changed.
+
+This new capability is also configurable using the new `format_noneditable_selector` option.
+
+For details see xref:content-formatting.adoc#format_noneditable_selector[Content Formatting].
+
+=== The autocompleter now supports a multiple character trigger using the new trigger configuration.
+
+Previously, the autocompleter only supported a single character trigger.
+
+With this update, the autocompleter now supports multi-character triggers.
+
+The new xref:mergetags.adoc[Merge Tags] Premium plugin, for example, uses the string `+{{+` as its default trigger.
+
+=== Some inline formatting, such as color and font size, can now be applied to list item elements
+
+In {productname} 6.2 list bullets and list numbers can be inline-formatted.
+
+If the contents of a list item are selected and inline formatting is applied — such as a change of color or change of font-size — this spot-formatting is also applied to the list item’s associated bullet or number.
```

Comment:
I know it's defined in the link, but it seems like we just pull `spot-formatting` out of nowhere here? Should it just be `inline-formatting` like it is in the rest of the point. Also, we have a bit of inconsistent hyphenation with inline formatting.

Reply:
> but it seems like we just pull spot-formatting out of nowhere here?

Inline-formatting is a web-developer–centric term-of-art. _Spot-formatting_ is the equivalent term used in the editorial world for this sort of ad-hoc formatting.